client/webserver/passbytes.go

```diff
@@ -0,0 +1,188 @@
+// This code is available on the terms of the project LICENSE.md file,
```

Comment:
`PassBytes` will have wider utility beyond **webserver**. How about moving it to **encode**?

Reply:
Great point.  We'll want to use it in server/admin for one.